### PR TITLE
Fix: Implement sat_track script and refactor launch script flow

### PR DESCRIPTION
**Pull request by jjjaspher**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix https://github.com/hlxsites/accenture-newsroom/issues/495
- https://github.com/hlxsites/accenture-newsroom/issues/495

Test URLs:
- Before: https://main--accenture-newsroom--hlxsites.hlx.page/
- After: https://load-beacon--accenture-newsroom--hlxsites.hlx.page/

### CODE REVIEW

File: scripts/scripts.js [hunk 1]
```diff
@@ -673,14 +673,37 @@ async function loadEager(doc) {
...
+
 async function loadJQueryDateRangePicker() {
   const filterInput = document.querySelector('#newslist-filter-input');
   if (!filterInput) {
     return;
   }
   // await import('./moment.min.js');
   await loadScript('/scripts/moment.min.js');
-  await loadScript('/scripts/jquery-3.5.1.min.js');
+  // await loadScript('/scripts/jquery-3.5.1.min.js'); // old
```
Comment (raymond-h-dev) [inline scripts/scripts.js:705]: remove the commented out codes

File: scripts/one-trust-geo-script.js [hunk 2]
```diff
@@ -1,15 +1,50 @@
...
+    const sSatTrackCookie = getCookie('sat_track');
+
+    if (typeof sSatTrackCookie !== 'undefined') {
+      return;
+    }
+
+    // list of geos where only strictly necessary cookies are allowed
+    const aSNGeos = ["GB", "BE", "IE", "FR", "RU", "ES", "IT", "DE", "AT", "CH"];
+    if (aSNGeos.indexOf(window.otUserLocation) !== -1) {
+      document.cookie = "sat_track=false;path=/;secure;max-age=31536000";
```
Comment (raymond-h-dev) [inline scripts/one-trust-geo-script.js:29]: are we sure about the max age values?

File: scripts/scripts.js (hunk 1 above)
Comment (jjjaspher) [reply scripts/scripts.js]: Updated, commented out codes removed

File: scripts/one-trust-geo-script.js (hunk 2 above)
Comment (jjjaspher) [reply scripts/one-trust-geo-script.js:29]: As per analytics team, that's the correct value.

**Review by raymond-h-dev — APPROVED** [review]

**Pull request merged**